ui/panels-plugin/src/plugins/line-chart/tooltip/SeriesInfo.tsx

```diff
@@ -11,59 +11,101 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-import { Box } from '@mui/material';
+import { Box, Divider, Stack } from '@mui/material';
 import SeriesMarker from './SeriesMarker';
 
 interface SeriesInfoProps {
   seriesName: string;
   y: number;
   markerColor: string;
   totalSeries: number;
+  wrapLabels?: boolean;
 }
 
 function SeriesInfo(props: SeriesInfoProps) {
-  const { seriesName, y, markerColor, totalSeries } = props;
+  const { seriesName, y, markerColor, totalSeries, wrapLabels } = props;
   // TODO (sjcobb): regex to remove __name__ and quotes, replace = with :
   if (totalSeries === 1) {
     return (
-      <Box sx={{ margin: '0 0 4px' }}>
+      <Stack spacing={0.5}>
         <Box
           sx={{
             height: '16px',
-            margin: '0 0 2px',
             display: 'flex',
             flexDirection: 'row',
             alignItems: 'center',
             justifyContent: 'left',
           }}
         >
           <SeriesMarker markerColor={markerColor} />
-          <Box component="span">value: {y}</Box>
+          <Box component="span" color="grey.300">
```

Comment:
Wouldn't be a good idea to have these colors not harcoded ? I'm thinking when we will have a theme in place, these hardcoded color could be an issue no ?

Reply:
I'll take care of this, I have a local branch where I started playing around with dark/light theme